quickstarts/microsoft.kusto/kusto-event-hub/main.bicep

```diff
@@ -0,0 +1,111 @@
+@description('Location for all resources')
+param location string = resourceGroup().location
+@description('Name of the cluster')
+param clusterName string = 'kusto${uniqueString(resourceGroup().id)}'
+@description('Name of the database')
+param databaseName string = 'kustodb'
+@description('Name of Event Hub\'s namespace')
+param eventHubNamespaceName string = 'eventHub${uniqueString(resourceGroup().id)}'
+@description('Name of Event Hub')
+param eventHubName string = 'kustoHub'
+
+resource eventHubNamespace 'Microsoft.EventHub/namespaces@2021-11-01' = {
+  name: eventHubNamespaceName
+  location: location
+  sku: {
+    capacity: 1
+    name: 'Standard'
+    tier: 'Standard'
+  }
+  properties: {}
+
+  resource eventHub 'eventhubs' = {
+    name: eventHubName
+    properties: {
+      messageRetentionInDays: 2
+      partitionCount: 2
+    }
+
+    resource kustoConsumerGroup 'consumergroups' = {
+      name: 'kustoConsumerGroup'
+      properties: {}
+    }
+  }
+}
+
+resource cluster 'Microsoft.Kusto/clusters@2022-02-01' = {
+  name: clusterName
+  location: location
+  sku: {
+    'name': 'Standard_D12_v2'
```

Comment:
make this a parameter (and set the defaultValue)

Reply:
Done.